### ISSUE TITLE
Running `prisma migrate up --experimental` results in relation "_Migration" already exists

### ISSUE THREAD

**Issue by vedantroy** [prisma/migrate#609]
## Bug description

When I run `prisma migrate up --experimental`, I get the error: relation _Migration already exists.

## How to reproduce

Use the following schema.prisma and then run:

```
prisma migrate save --experimental
prisma migrate up --experimental
```

## Expected behavior

This doesn't happen.

## Prisma information

The schema:

```
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("PGURL")
}

model FooBar {
  a         String                       @id
  b  String
  c String
  d       String?
  e       Type1?
  f     Type2?
  g  Json?
  h  DateTime?
  i  DateTime?
  j  DateTime?
  k String?

  @@map(name: "foo_bar")
}

enum Type1 {
  APPROVED
  DRAFT
  REJECTED
}

enum Type2 {
  Attribute
  List
}
```

## Environment & setup

- OS: Linux
- Database: postgres
- Node.js version: 12.18.1
- Prisma version:
```
@prisma/cli          : 2.8.1
Current platform     : debian-openssl-1.1.x
Query Engine         : query-engine 439da16b2f8314c6faca7d2dad2cdcf0732e8a9c (at node_modules/@prisma/cli/query-engine-debian-openssl-1.1.x)
Migration Engine     : migration-engine-cli 439da16b2f8314c6faca7d2dad2cdcf0732e8a9c (at node_modules/@prisma/cli/migration-engine-debian-openssl-1.1.x)
Introspection Engine : introspection-core 439da16b2f8314c6faca7d2dad2cdcf0732e8a9c (at node_modules/@prisma/cli/introspection-engine-debian-openssl-1.1.x)
Format Binary        : prisma-fmt 439da16b2f8314c6faca7d2dad2cdcf0732e8a9c (at node_modules/@prisma/cli/prisma-fmt-debian-openssl-1.1.x)
Studio               : 0.296.0
```

**Comment by VaivanOmmeren** [comment]
Running into the same issue here.

If I were to copy the code generated in the `README.md` in the generated migration it also runs into errors.

The command `CREATE TYPE "public"."PipelineType"` would only work if I change it to `CREATE TYPE "public".PipelineType`

Other than that the migration somehow does get saved in the database however the `_Migration` table is not present.
I become unable to generate a new migration for that database in another project because it does see the previously run migration, I am unsure as to how it know seeing as the database still is completely empty.
Dropping the database and creating a new one does give me the option to create a new migration but does not fix the issue of getting `relation _Migration already exists`

**Comment by thortek** [comment]
I have the exact same issue running Mac OS 10.15.6 using postgres with Node 12.13.0 and Prisma CLI and Client at 2.9.0

**Comment by tomhoule** [comment]
I just tried to reproduce this issue with both 2.10.0-dev.21 and 2.9.0, on sqlite and postgres, and could not trigger the problem.

Additional info:

```
erewhon.tom.~/tmp/migrate-bug-confirmatino λ uname -a
Linux erewhon 5.4.0-48-generic #52-Ubuntu SMP Thu Sep 10 10:58:49 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
``` 

postgres version: 10

**Comment by vedantroy** [comment]
@tomhoule  - I think a recent update fixed the problem?

**Comment by thortek** [comment]
I was able to resolve this issue with "TRUNCATE _Migration" on my Postgres instance running in Docker.

**Comment by tomhoule** [comment]
Thanks for taking the time to report this issue!

We've recently released a Prisma Migrate Preview ([2.1.3.0 release notes](https://github.com/prisma/prisma/releases/tag/2.13.0)), which has quite a few changes compared to the previous experimental version. We believe this issue is no longer relevant in this new version, so we are closing this.

We would encourage you to try out Prisma Migrate Preview. If you come across this or any other issue in the preview version, please feel free to open a [new issue in prisma/prisma](https://github.com/prisma/prisma/issues/new/choose).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue](https://github.com/prisma/prisma/issues/4531).
